### ISSUE TITLE
[Track2] The generated package.json doesn't include "lint" in script

### ISSUE THREAD

**Issue by dw511214992** [Azure/autorest.typescript#1069]
This error is reported by pipeline:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=964536&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=aea0e0e1-1e00-5c1b-bba8-b2b238976da3
I make a fix manaully by adding:
```
"lint": "echo skipped"
```
